### PR TITLE
add 08 DefinePlugin advanced

### PR DESCRIPTION
**Pull request by Foxandxss**

### CODE REVIEW

File: 03 Misc/08 DefinePlugin advanced/src/averageService.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function getAvg(scores) {
```
Comment (brauliodiez) [inline 03 Misc/08 DefinePlugin advanced/src/averageService.js]: Should we remove averageService and display total components and only focus on the new student list component? (just to remove noise from the sample)

File: 03 Misc/08 DefinePlugin advanced/src/totalScoreComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```
Comment (brauliodiez) [inline 03 Misc/08 DefinePlugin advanced/src/totalScoreComponent.jsx]: Same here i think I would remove this component and only focus on the student list

**Comment by Foxandxss** [comment]
Why not, we can remove both of them. I was just going to say "We have a student app with nice averages but not a list of students, so we are going to fix that". But I agree that we could reduce the noise.

**Comment by Foxandxss** [comment]
Okey, I removed them.

**Comment by brauliodiez** [comment]
Good stuff, I have merged it.

**Pull request merged**